pep-0020.txt

```diff
@@ -48,29 +45,20 @@ The Zen of Python
 Easter Egg
 ==========
 
-::
+.. code-block:: pycon
```

Comment:
`pycon`?  Is that a typo?

Reply:
[Nope](https://pygments.org/docs/lexers/#pygments.lexers.python.PythonConsoleLexer)

Reply:
Hah!  That's not confusing at all :smile: